tests/test_smoke.py

```diff
@@ -1945,6 +1999,10 @@ def cli_ls_cmd(store_type, bucket_name, suffix=''):
             else:
                 url = f's3://{bucket_name}'
             return f'AWS_SHARED_CREDENTIALS_FILE={cloudflare.R2_CREDENTIALS_PATH} aws s3 ls {url} --endpoint {endpoint_url} --profile=r2'
+        if store_type == storage_lib.StoreType.IBM:
+            bucket_rclone_profile = Rclone.get_rclone_bucket_profile(
+                bucket_name, Rclone.RcloneClouds.IBM)
+            return f'rclone ls {bucket_rclone_profile}:{bucket_name}/{suffix}'
```

Comment:
Can we set `if .. else ..` statement on when `suffix` is given and not given?

Reply:
Since there isn't a dialog box for the error above, I'll respond here:
It seems that `curl` and `sudo` are missing in your local environment, hence `rclone` installation fails. Can't I rely on users having access to these out of the box tools?

Reply:
> Why? if it isn't specified it's an empty string, which is considered to be an empty path.

This is to maintain consistency with structure of this function.

Reply:
> Since there isn't a dialog box for the error above, I'll respond here:
It seems that curl and sudo are missing in your local environment, hence rclone installation fails. Can't I rely on users having access to these out of the box tools?

In that case, let's have `rclone` set up in `setup.py L132` as an IBM dependency. It makes more sense to have `rclone` installed when environment is being setup. I should have came up with this conclusion earlier, sorry about the hassle.

Reply:
Following a discussion on the topic with @romilbhardwaj:
- rclone will be added as a manual installation requirement to [docs](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html#ibm). 
- If rclone isn't detected on runtime an exception will be raised early on.